### DISCUSSION TITLE
How to add/update value "Creator earnings" when using openseaSdk on testnet

### DISCUSSION THREAD

**Discussion by lehongvo** [ProjectOpenSea/opensea-js#1247] [Q&A]
### Component

API

### Describe the feature you would like

We want to using openseaSDK to add/update creator earnings like this
<img width="1048" alt="image" src="https://github.com/ProjectOpenSea/opensea-js/assets/71160314/bba5c8a1-6b21-4905-ac46-7bbcca7ae798">
=

**Comment by ryanio** [comment]
~~When you deploy the contract if you have contractURI defined we will set the creator earnings, as described on this page https://docs.opensea.io/docs/contract-level-metadata~~

~~After the contract is already deployed there is no way to update it via API, you must use the OpenSea interface.~~

**Reply by lehongvo** [reply]
@ryanio Can you give us a sample metadata link?

**Reply by ryanio** [reply]
@lehongvo sorry I checked with my team and this is not possible anymore. You can use the Royalty Registry to set it and we will respect that: https://royaltyregistry.xyz/
